File: ejercicios-js/ejercicio3.js
const palabras = [
  "Aceituna", "Murciélago", "Educación", "Aeropuerto", "Otorrinolaringólogo",
  "Euforia", "Aceite", "Paleontólogo", "Arquitectura", "Hipopótamo"
];

// Elegir una palabra aleatoria
const palabraAleatoria = palabras[Math.floor(Math.random() * palabras.length)];

// Definir conjunto de vocales (con tildes incluidas)
const vocalesValidas = new Set("aeiouáéíóú");

// Extraer vocales de la palabra (ignorando mayúsculas)
const vocalesEncontradas = Array.from(palabraAleatoria.toLowerCase())
  .filter(letra => vocalesValidas.has(letra));

// Mostrar resultados
console.log(" Palabra seleccionada:", palabraAleatoria);
console.log(" Vocales encontradas:", vocalesEncontradas.join(", "));
console.log(" Total de vocales:", vocalesEncontradas.length);

File: ejercicios-py/ejercicio3.py
""" HAZ UNA FUNCION QUE CUENTE EL NUMERO DE VOCALES EN UN STRING PASADA POR PARAMETROS"""

lista_vocales = 
["a" , "e", "i", "o","u"]




def cont_vocales(msg):
    nvocales =0
    for char in msg:
        
        if char in lista_vocales:
            nvocales +=1
        
        print (f"{msg} tiene {nvocales} vocales.")

cont_vocales("Aceituna")
cont_vocales("Murciélago")
cont_vocales("Educación")
cont_vocales("Aeropuerto")
cont_vocales("Otorrinolaringólogo")
cont_vocales("Euforia")
cont_vocales("Aceite")
cont_vocales("Paleontólogo")
cont_vocales("Arquitectura")
cont_vocales("Hipopótamo")
cont_vocales("Aceituna")
cont_vocales("Aceituna")
        